product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -0,0 +1,510 @@
+---
+title: Performing a Postgres major version rolling upgrade 
+---
+
+## Performing a Postgres major version rolling upgrade
+
+This document outlines the procedure for performing a major version upgrade of a Postgres cluster using Postgres Distributed (PGD) as installed by Trusted Postgres Architect (TPA). The rolling upgrade process allows updating individual cluster nodes to a new major Postgres version while maintaining cluster availability and operational continuity. This approach minimizes downtime and ensures data integrity by allowing the rest of the cluster to remain operational as each node is upgraded sequentially. Upgrading your Postgres cluster to a newer major version is recommended to access improved features, performance enhancements, and security updates. This guide provides step-by-step instructions to facilitate a smooth and controlled upgrade process.
+
+!!! Note
+Ensure that your databases and configuration files are backed up before starting the upgrade process to help prevent data loss.
+!!!
+
+To perform the rolling upgrade of the Postgres major version on your PGD cluster, carry out the following steps on each node in the cluster:
+
+- Confirm the current Postgres version.
```

Comment:
That's a bare heading. At least provide some direction for how you would do this.
"Versions can be viewed from PGD by logging in and running: `select * from bdr.group_version_details;`"